mbof/views.py

```diff
@@ -26,6 +27,15 @@ class EventViewSet(viewsets.ModelViewSet):
     """
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
+    
+
+class CurrentEventViewSet(EventViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
+    """
+    current_time = timezone.now()
+    queryset = Event.objects.filter(startTime__lte=current_time).filter(endTime__gte=current_time)\
```

Comment:
Also the backslash for continuation at the end of the line is annoying, isn't it?  Is it really required?  (I'm guessing that you probably wouldn't have used it here if you hadn't gotten an error without it.)

I think that if you want to break the line and you hate the backslash as much as I do, you can get around it by enclosing the whole expression in parentheses:

```python
    queryset = (Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time)
                .order_by('postingTime'))
```